tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -72,10 +68,63 @@ def test_createami(region, os, instance, request, pcluster_config_reader, vpc_st
         any("packages.chef.io".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_false()
     assert_that(
-        any("Thank you for installing Chef".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any(
+            "Thank you for installing Cinc Client".lower() in s.lower()
+            for s in pcluster_createami_result.stdout.split("\n")
+        )
     ).is_true()
     assert_that(
-        any("Starting Chef Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any("Starting Cinc Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_true()
 
     assert_that(pcluster_createami_result.stdout).does_not_contain("No custom AMI created")
+
+
+@pytest.mark.dimensions("us-west-2", "c5.xlarge", "centos7", "*")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu1804", "*")
+@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "*")
+def test_createami_post_install(
+    region, os, instance, test_datadir, request, pcluster_config_reader, vpc_stack, architecture
+):
+    """Test post install script and base AMI is ParallelCluster AMI"""
+    cluster_config = pcluster_config_reader()
+
+    # Get ParallelCluster AMI as base AMI
+    ami_type = "pcluster"
```

Comment:
Minor: You can avoid to define this variable given the fact you're using it in a single place and the meaning will be clear when passing to ami_type parameter by using the keyword.

```python
retrieve_latest_ami(region, os, ami_type="pcluster",...
```

Reply:
Done